src/template.js

```diff
@@ -15,15 +15,29 @@ export function readFileFromPath(path){
         content = fs.readFileSync(path, "utf8");
     } catch (err) {
         if (err.code === "ENOENT") {
-            logger.warn("Error: file not found: " + path);
-            return "Error: file not found: " + path;
+            return null;
         } else {
             throw err;
         }
     }
     return content;
 }
 
+/** 
+ * Sync file read with url check 
+ * @param {string} url 
+ * @return {string} 
+ */
+export function readFileFromURL(url) {
+    var content;
+    try {
+        content = request("GET", url).getBody("utf8");
```

Comment:
It never timeout.

- [sync-request](https://github.com/ForbesLindesay/sync-request "sync-request")